common/lib/xmodule/xmodule/modulestore/tests/test_cross_modulestore_import_export.py

```diff
@@ -337,6 +338,7 @@ def asset_collection(self):
 )
 MODULESTORE_SETUPS = DIRECT_MODULESTORE_SETUPS + MIXED_MODULESTORE_SETUPS
 MODULESTORE_SHORTNAMES = DIRECT_MS_SETUPS_SHORT + MIXED_MS_SETUPS_SHORT
+SPLIT_MODULESTORE_SETUP = (MixedModulestoreBuilder([('split', VersioningModulestoreBuilder())]),)
```

Comment:
Since you've defined this, move up to above `MIXED_MODULESTORE_SETUPS` and use this new definition in that list.